### DISCUSSION TITLE
How can I stop receiving notifications from myself in Github

### DISCUSSION THREAD

**Discussion by nmalzieu** [community/community#56950] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello !

I'm using Github notifications and I always receive notifications for stuff that I do myself, for instance pushing to a repository.
It shows in the notification center in the category "author": 

<img width="567" alt="Capture d’écran 2023-06-01 à 16 36 58" src="https://github.com/community/community/assets/2102342/4c5ae133-4160-4bbd-8337-c591217de1fd">

I know that I can create a filter on the left side for this, but I want my global inbox to not show stuff from myself!
Is that possible?

**Comment by Gustavonn07** [comment]
See if you watched it in your own project, this function always allows you to receive an update whenever the repository is updated

![image](https://github.com/community/community/assets/84361085/8bf53fd1-d5d6-4d54-bb03-a256c9e8178f)

-------------------------------------------------------------------------------------------------------------------------------------------------

Once inside:
![image](https://github.com/community/community/assets/84361085/f0626131-b02d-4b42-b8fa-709b150da023)

**Reply by nmalzieu** [reply]
Hi, I receive notifications for stuff I do (my own commits)
I want to he able to receive notifications for stuff other people do but not me! How is your solution helping me to do that?

**Comment by DSDmark** [comment]
Hi @nmalzieu,

The another way to do that. You can off your notification inside the setting like this:

just click on the settings and inside that you can change notification setting:

![image](https://github.com/community/community/assets/85744816/a9a7da28-eed1-4bba-86b9-ac21c9b53872)

here you can change:

- Automatically watch repositories
- Default notifications email
- Actions Notifications  
- and lots of other setting, I guess you like to change.

![image](https://github.com/community/community/assets/85744816/b4ad27fb-375a-485b-9b90-1f8aa7ce8d78)

Here is the direct like:- [**notification settings**](https://github.com/settings/notifications)

**Reply by nmalzieu** [reply]
Hi, I receive notifications for stuff I do (my own commits)
I want to he able to receive notifications for stuff other people do but not me! How is your solution helping me to do that?

**Reply by DSDmark** [reply]
Just go and explore these settings which I describe above, you can find automatically how it is help.

And also you change **watching** if you don't want to receive notification when you push or commit, you repo:

![image](https://github.com/community/community/assets/85744816/3f143783-8022-499b-8d59-76fed236ac32)

---
![image](https://github.com/community/community/assets/85744816/d2e2c4b3-e6a0-4aae-a56e-bd0b215ff0ee)

here is direct link: https://github.com/watching

**Reply by Gustavonn07** [reply]
I would say that following @DSDmark  step by step would work

**Reply by Biserkov** [reply]
**tl;dr:** Imagine a chat app. I want to receive notifications when _others_ message the group chat. I most definitely do **not** want to receive notifications when it is _I_ who is sending the message to the group chat.

Both @Gustavonn07 and @DSDmark are misunderstanding the issue. Let me try to explain (I have the issue myself).

When I create a PR in our team repository, I get notification that _I_ created a PR. This is unnecessary. I know that _I_ created a PR - I created it.

Same thing when I merge a PR. I get a notification that _I_ merged a PR. This is unnecessary. I know that _I_ merged a PR - I merged it.

The solutions you proposed would stop me from receiving any notifications about the repository. But I want notifications when _others_ create or merge PRs. But not when _I_ do it.

P.S. This also happens when I assign _myself_ to a PR. So I'm getting 3 extra notifications per PR.

**Reply by phansch** [reply]
I'm also annoyed by this and it makes the GitHub notifications inbox almost unusable.
The only workaround is to receive email notifications and build your own filters to ignore things you did yourself.

**Comment by snaffi** [comment]
@nmalzieu 

Looks like you could exclude that emails in notifications settings

https://github.com/isaacs/github/issues/957

<img width="1030" alt="image" src="https://github.com/community/community/assets/1266395/edd91ea4-6c80-4d37-b588-563d67b0aeff">

**Reply by signalogic** [reply]
this doesn't fix it. I still receive e-mail notification of my own updates

**Reply by fcFn** [reply]
Also this appears to only apply to emails and not the GH inbox.

**Comment by fcFn** [comment]
It should probably allow you to configure which notifications to send based on the way the request was authorized. I.e., some might want to log automated things (though I suppose that should usually be done via a bot) but not things they did manually on the website or elsewhere.
